chatty/socket.py

```diff
@@ -1,77 +1,82 @@
 from .evented import Evented
-from ws4py.client.tornadoclient import TornadoWebSocketClient
-from tornado import ioloop
-import random
-import json
 
-# A ws4py websocket wrapped in an Evented class.
-class EventedWebSocket(TornadoWebSocketClient, Evented):
-    def __init__(self, *args, **kwargs):
-        Evented.__init__(self)
-        TornadoWebSocketClient.__init__(self, *args, **kwargs)
+from tornado.websocket import websocket_connect
+from tornado.ioloop import IOLoop
```

Comment:
Fair enough. I might work on that Soon™. :smile: 
I also noticed the [`rewrite`](https://github.com/WatchBeam/beam-client-python/tree/rewritten) branch - would you like me to merge it in with what I just wrote?